packages/web/docs/src/app/blog/(posts)/introducing-mcp-graphql/page.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: 'Introducing: GraphQL MCP server'
+authors: blurrah
+tags: [graphql, mcp]
```

Comment:
```suggestion
tags: [graphql, ai]
```
If we want to limit the number of tags, ai would be better to use than mcp since we can attribute more articles to that.